cmd/tape/main.go

```diff
@@ -115,6 +117,16 @@ func checkArgs(rate *float64, burst, signerNumber, parallel *int, con string, lo
 		fmt.Printf("As rate %d is bigger than burst %d, real rate is burst\n", int64(*rate), int64(*burst))
 	}
 
+	// enable prometheus but not provide --prometheus-addr option, use default prometheus address ":8080"
+	if enablePrometheus && len(*prometheusAddr) == 0 {
+		*prometheusAddr = DEFAULT_PROMETHEUS_ADDR
```

Comment:
we should add a log here as hit for default port.

Reply:
How about adding prometheus startup log?
```shell
[INFO] prometheus running at :9090
```